### DISCUSSION TITLE
How to know all available instruments in alda?

### DISCUSSION THREAD

**Discussion by WingDust** [alda-lang/alda#470] [Q&A] [answered]
How to know all available instruments in alda? like piano ....

**Comment by daveyarwood** [comment, marked as answer]
You can run `alda instruments` to print the list of all available instruments.

Many of the instrument names (e.g. `midi-acoustic-grand-piano`, `midi-harmonica`) have shorter aliases (e.g. `piano`, `harmonica`). There isn't currently a good way to see all of the aliases, unfortunately, apart from looking at the source code or trial and error.
